api_v1/openapi.yml

```
@@ -38,6 +38,7 @@ info:
 
 servers:
   - url: "{{ site.url }}/api/v1"
+    description: Production
```

To differentiate from our older API.

```suggestion
    description: Production v1
```

---

Good thinking!